src/styles/blocks/container.scss

```diff
@@ -0,0 +1,14 @@
+.container {
+  box-sizing: content-box;
```

Comment:
It is not perfect having different value for box-sizing, unless you have a reason to